src/js/components/Accordion/stories/Custom.js

```diff
@@ -7,7 +7,12 @@ import { Accordion, AccordionPanel, Box, Grommet, Text } from 'grommet';
 
 const CustomAccordionTheme = {
   accordion: {
-    heading: { level: '3' },
+    heading: {
+      level: 3,
+      hover: {
```

Comment:
I'm not sure if it is semantically correct to place a hover on a heading element vs the interactive element. Is there a reason the `hover` is nested with `heading` and not directly under accordion?

Reply:
With the current code `heading` is used under the hood for the accordion I was just thinking of a way to get rid of the hover since it is already built in and not defined in the theme. I can move to be accordion.hover. I think my original thought process was this hover effects the heading size in the accordion